PlasticEnemy/Data/Entities/User.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
```

Comment:
Why do we need this using?

Reply:
Didn't tidy up usings before commit, will remove